translations/zh/built-in-makers-zh.md

```diff
@@ -0,0 +1,80 @@
+内置制造机 Built In Makers
+===============
+下面是一个Fili内置标准制造机表格。对于标准制造机，请参考[Configuring Metrics][configuring metrics]。
```

Comment:
[Configuring Metrics][configuring metrics] is not rendered correctly. please look at the bottom section of this file where I left a comment